bin/convert.py

```diff
@@ -30,7 +30,9 @@
                 print("no biolinkType, default to NameThing")
                 categories = ["biolink:NamedThing"]
             ty = column_mapping["type"]
-            idt = old_identifiers[table].get(column, 'DUMMY:000000')
+            idt = old_identifiers[table].get(column)
```

Comment:
Just for my knowledge (and a comment could potentially be added here in the code), what values are being returned here that was breaking the old code?

Reply:
@maximusunc The old code did not set ```idt``` to DUMMY when the column exists with a value None which happens to occur for many column variables that don't have a linked identifier. So I updated it to get it first, then check if it is not an empty value or None, which works for PCD and covid instances when I tried to create a mapping file for them. With the old code, linked identifiers are set as None or empty string, which triggered the exception from ICEES API code which does not allow empty identifiers.